handler/interceptors.go

```diff
@@ -82,6 +83,10 @@ func NewGrpcErrorf(code codes.Code, format string, args ...interface{}) *GrpcErr
 	}
 }
 
+const (
+	START_PROCESSING_ANY_REQUEST = 1
```

Comment:
It probably makes sense to move these constants into `configuration_service` to use it in `StopProcessingRequests` and `StartProcessingRequests` methods